internal/evaluator/conftest_evaluator.go

```diff
@@ -103,8 +103,29 @@ func trim(results *[]Outcome) {
 		return trimmed
 	}
 
+	addNote := func(results []Result) []Result {
+		for i := range results {
+			var description, code string
+			var ok bool
+			if description, ok = results[i].Metadata[metadataDescription].(string); !ok {
+				continue
+			}
+
+			if code, ok = results[i].Metadata[metadataCode].(string); !ok {
+				continue
+			}
+
+			if term, ok := results[i].Metadata[metadataTerm]; ok {
+				code = fmt.Sprintf("%s:%s", code, term)
+			}
+			results[i].Metadata[metadataDescription] = fmt.Sprintf("%s. To exclude this rule add %q to the `exclude` section of the policy configuration", strings.TrimSuffix(description, "."), code)
+		}
+
+		return results
```

Comment:
No need for this function to return a slice of results since it is modifying the one that is given? Alternatively, we could make it so it creates a copy (which IMO is always safer).

Reply:
Can't compose it below otherwise